### ISSUE TITLE
Portable Installation

### ISSUE THREAD

**Issue by TheBlubb14** [AntMeNet/AntMeClassic#41]
For school its most of the time much more easier to run a program of the usb stick than to install it on the computer.
Also i like portable installations more than having to use an installer and not exactly knowing what the installer does.

So we should additionally include a portable installation folder in our CI, which will contain a build and out of the box runnable version of AntMe

**Comment by tomwendel** [comment]
Good idea, but a bit tricky. The problem is that there is a hard reference to the SimulationCore.dll which sits in the installation folder. when the player generates a new ant project, this reference is fixes and need to be changed manually when the position changes.

That means if the player starts the game from a different location the ant will not build anymore.

**Comment by TheBlubb14** [comment]
We could maybe setup a custom local NuGet feed. 
Then in the installation folder we have a folder which acts as a source to the NuGet packages and in the generated ant project we have a `Nuget.Config` which points to the folder.
For each AntMe Update, we would just add a new NuGet package to the folder and existing ant projects can easily be upgraded, by just updating the new AntMe version

**Comment by tomwendel** [comment]
Even better: publish SimulationCore assembly as a public Nuget package which can be consumed by the application itself :)
But then we still have the problem of the executable path which is part of the startup parameters of the project. Ant projects are only class libraries which need a startup exe and its own location as a parameter to have the F5 experience.
